src/components/App.js

```diff
@@ -23,6 +22,17 @@ export default class App extends React.Component {
     error: PropTypes.object,
   };
 
+  getSnapshotBeforeUpdate() {
+    const { error } = this.props;
+
+    if (error) {
+      notification['error']({
+        message: 'Login Failed',
```

Comment:
THis is odd. Shouldn't it be a generic expression like "Something went wrong".
Or even better would be if there was an `errorTitle` and `errorDescription` or something.

Reply:
The error here is specifically a login error. I should rename the variable.